README.md

```diff
@@ -6,118 +6,131 @@
+
+   The example client program uses the environment variable `GT_STRUCTURE_ID` to determine which Structure to run.
+   Set this environment variable to the Structure ID you registered in the previous step.
+
+   ```bash
+   export GT_STRUCTURE_ID={STRUCTURE_ID}
+   ```
+
+   Or you can register and set the environment variable in one step.
+
+   ```bash
+   export GT_STRUCTURE_ID=$(gt skatepark register --structure-config-file structure_config.yaml --tldr)
+   ```
 
 > [!IMPORTANT]
-> Structures registered with the Skatepark are not persisted across restarts. You will need to re-register the Structure each time you restart Skatepark.
-6. Confirm that the Structure is registered.
+> Structures registered with the Skatepark are not persisted across restarts. You will need to re-register the Structure each time you restart Skatepark. 6. Confirm that the Structure is registered.
+
     ```bash
     gt skatepark list
     ```
     You should see a list of registered Structures and the directories they point to, confirming that your Structure was properly registered
-7. You can load environment variables into your Structure by creating an `.env` file in the directory of the Structure you registered. 
-    1. Create a file named `.env` in the same directory that your Structure code is in.
-    2. Open the `.env` file in a text editor.
-    3. Copy the contents of `.env.structure.example` into the `.env` file. This specifies all of the values necessary and explains how they are used.
-    3. The template expects an `OPENAI_API_KEY` environment variable by default to function. Add OPENAI_API_KEY=_your OpenAI API Key here_ to the `.env` file and save it.
-    4. As you expand on the template, you may add any other environment variables your Structure depends on to this file.
-8. Rebuild the structure to load in the new environment variable. 
-    Note that this is only required for changes to `.env` or `requirements.txt`. Code changes do not require a rebuild. 
-    ```bash
-    gt skatepark build
-    ```
+
+7. You can load environment variables into your Structure by creating an `.env` file in the directory of the Structure you registered.
+   1. Create a file named `.env` in the same directory that your Structure code is in.
+   2. Open the `.env` file in a text editor.
+   3. Copy the contents of `.env.structure.example` into the `.env` file. This specifies all of the values necessary and explains how they are used.
+   4. The template expects an `OPENAI_API_KEY` environment variable by default to function. Add OPENAI*API_KEY=\_your OpenAI API Key here* to the `.env` file and save it.
+   5. As you expand on the template, you may add any other environment variables your Structure depends on to this file.
+8. Rebuild the structure to load in the new environment variable.
+   Note that this is only required for changes to `.env` or `requirements.txt`. Code changes do not require a rebuild.
+   ```bash
+   gt skatepark build
+   ```
```

Comment:
did anything actually change here or is this just the formatter?

Reply:
Just the formatter here, and also a markdown linter extension